js/src/forum/utils/UserControls.js

```diff
@@ -124,13 +124,18 @@ export default {
    * @param {string} type
    */
   showDeletionAlert(user, type) {
-    const { username, email } = user.data.attributes;
     const message = {
       success: 'core.forum.user_controls.delete_success_message',
       error: 'core.forum.user_controls.delete_error_message',
     }[type];
 
-    app.alerts.show({ type }, app.translator.trans(message, { username, email }));
+    app.alerts.show(
+      { type },
+      app.translator.trans(message, {
+        user,
```

Comment:
isn't this meant to be `username: user.displayName()` ?

Reply:
It's now using our "magic" translator behavior that automatically provides a `username` variable with the output of the `username()` helper function if you pass a user (which actually returns the display name). This is actually more consistent with the other uses of the translator.

Reply:
oh wow, I actually wasn't aware it was a thing lol